test/test_rospy/test/rostest/test_latch_unsubscribe.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+# Software License Agreement (BSD License)
+#
+# Copyright (c) 2008, Willow Garage, Inc.
```

Comment:
I guess this should be different since the code is not coming from Willow Garage.